apps/meteor/app/livechat/server/lib/LivechatTyped.ts

```diff
@@ -353,11 +353,12 @@ class LivechatClass {
 			throw new Meteor.Error('error-omnichannel-is-disabled');
 		}
 
-		if (await LivechatContacts.isChannelBlocked(this.makeVisitorAssociation(visitor._id, roomInfo.source))) {
+		const visitorAssociation = this.makeVisitorAssociation(visitor._id, roomInfo.source);
```

Comment:
Is this part of the fix or is another bug? 👀

Reply:
Hmm it's part of the fix. I just split this into a new var since it'll be used in the next line and reused by the callback